packages/suite/src/views/settings/general/Theme.tsx

```diff
@@ -61,15 +59,15 @@ export const Theme = () => {
 
     const selectedValue = getOption(autodetectTheme ? 'system' : getVariant(theme.variant));
 
-    const onChange = ({ value }: { value: SuiteThemeVariantOptions }) => {
+    const onChange = ({ value }: { value: SuiteThemeVariant }) => {
         if ((value === 'system') !== autodetectTheme) {
             setAutodetect({ theme: !autodetectTheme });
         }
-        if (value === 'system') {
-            window.desktopApi?.themeSystem();
-        } else {
+        if (value !== 'system') {
             setTheme(getVariant(value));
```

Comment:
I believe that after the other type fixes, the `getVariant` method becomes useless and it will be possible to replace its calls directly by the param value. Correct me if I'm wrong.